lib/Controller/UsersController.php

```diff
@@ -727,7 +727,7 @@ protected function sendNotificationMail($userId) {
 			$message = $this->mailer->createMessage();
 			$message->setTo([$email => $userId]);
 			$message->setSubject($this->l10n->t('%s password changed successfully', [$this->defaults->getName()]));
-			$message->setPlainBody($msg);
+			$message->setHtmlBody($msg);
```

Comment:
Please add the other template too :-)

Reply:
Added `setPlainBody` back.

Reply:
👍

Reply:
Misunderstood, in my earlier PR. Corrected back to have both plain text and html to make it multipart message.

Reply:
done 👍